src/util/internalCallTree.js

```diff
@@ -73,7 +74,7 @@ class InternalCallTree {
     var scopeId = util.findLowerBoundValue(vmtraceIndex, scopes)
     scopeId = this.scopeStarts[scopeId]
     var scope = this.scopes[scopeId]
-    while (scope.lastStep && scope.lastStep < vmtraceIndex) {
+    while (scope.lastStep && scope.lastStep < vmtraceIndex && scope.firstStep > 0) {
```

Comment:
Before we got out of the loop when `scope.lastStep === undefined` that would signify that we are at the root scope (with firstStep === 0).
That's better to check directly for  `firstStep > 0`.
More over I found a case where `lastStep` is not undefined and `firstStep` is 0.